GraphsV2/utils/customTooltip.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import ReactTooltip from "react-tooltip";
+
+import columnAccessor from "../../utils/columnAccessor";
+
+export default (properties) => {
+
+    let getTooltipContent;
+
+    const setTooltipAccessor = (tooltip, yTicksLabel) => {
+        if (!tooltip) {
+            return;
+        }
+
+        // This function is invoked to produce the content of a tooltip.
+        getTooltipContent = (d) => {
+            // The value of this.hoveredDatum should be set by subclasses
+            // on mouseEnter and mouseMove of visual marks
+            // to the data entry corresponding to the hovered mark.
+            if(d && !d.isChord) {
```

Comment:
customTooltip should accept a tooltipContent  function, please remove customization and checks on specific graph types.